tests/piqueserver/test_core_commands.py

```diff
@@ -2,9 +2,12 @@
 from unittest.mock import Mock
 
 from piqueserver import core_commands
+import piqueserver.core_commands.movement as Movement
```

Comment:
`Movement` is not pep8 compliant here. Module names are required to be all lower case. UpperCamelCase signifies a class.